lib/sles4sap_publiccloud.pm

```diff
@@ -831,4 +837,105 @@ sub list_cluster_nodes {
     die 'None of the hosts are currently part of existing cluster' unless @cluster_nodes;
 }
 
+=head2 get_hana_database_info
+
+    Parses "hdbsql" command output.
+    Returns 1 if hana database is online, 0 otherwise.
```

Comment:
This function is basically returning a boolean (1 if DB is online, 0 if is offline). As such, the function `get_hana_database_info` is not returning any information about the DB except whether it is up or down. I suggest changing the name of the function to reflect that: `is_hana_database_online()` or `get_hana_database_status` ... but if keeping 1 or 0, I think the first one is more correct.

Reply:
ok